app/scripts/question_tooltip.js

```diff
@@ -0,0 +1,134 @@
+export default () => [
+    '<style>',
+    '.tooltip {position: relative; display: inline-block;}',
+    '.tooltip .tooltiptext {visibility: hidden; width: 200px; background-color:rgb(69, 69, 69, 0.99);',
+    'color: #fff; text-align: center; border-radius: 6px; padding: 5px 0; position:',
+    'absolute; z-index: 1; bottom: 100%; left: 120%; margin-left: -60px; opacity: 0;',
+    'transition: opacity 0.3s;}',
+    '.tooltip .tooltiptext::after {position: relative; -index: 1; bottom: 100%; left: 50%; margin-left: -60px;',
+    'border-width: 5px; border-style: solid; border-color: #555 transparent transparent transparent;}',
+    '.tooltip:hover .tooltiptext {visibility: visible; opacity: 1;}',
+    '</style>',
+
+    '<div style="width: 100px; height: 100px; display: inline; padding-left: 85px;" id="check_true" class="tooltip">',
+    '<svg width="23" height="23" viewBox="0 0 23 23" fill="none" ',
+    'xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" ',
```

Comment:
This seems hard to maintain, understand and is yet code duplication since the `questionMark.svg` has the same data.

Could you check the possibility of using an `<img/>` tag? I know it has support for SVG. I recommend the read: https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web

Reply:
I recommend this reading about inserting images into chrome extension: https://stackoverflow.com/questions/10865789/embedding-images-into-a-chrome-extension

Reply:
We were able to use the img tag to use IMG.

Function used:
chrome.extension.getURL ("../ images / questionMark.svg")

For this it was necessary to add the SVG image in the manifest:
"web_accessible_resources": [
"images / questionMark.svg"
]